solutions/src/main/java/ru/sberbank/school/task02/CurrencyCalc.java

```diff
@@ -0,0 +1,70 @@
+package ru.sberbank.school.task02;
+
+
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+import ru.sberbank.school.task02.exception.*;
+
+import java.math.BigDecimal;
+import java.util.List;
+
+public class CurrencyCalc implements FxConversionService {
+
+    private ExternalQuotesService quotes;
+
+    public CurrencyCalc(ExternalQuotesService externalQuotes) {
+        this.quotes = externalQuotes;
+    }
+
+    @Override
+    public BigDecimal convert(ClientOperation operation, Symbol symbol, BigDecimal amount) {
+        if(operation == null){
+            throw new NullPointerException("operation missing");
+        }
+        if(amount == null){
+            throw new NullPointerException("amount missing");
+        }
+        if(symbol == null ){
```

Comment:
WhitespaceAround: &#39;if&#39; is not followed by whitespace.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a1050029608>*